src/main/src/main/java/org/geoserver/config/impl/GeoServerInfoImpl.java

```diff
@@ -281,6 +283,16 @@ public Boolean isXmlExternalEntitiesEnabled() {
         return this.xmlExternalEntitiesEnabled;
     }
 
+    @Override
+    public void setTrailingSlashMatch(Boolean trailingSlashMatch) {
+        this.trailingSlashMatch = trailingSlashMatch;
+    }
+
+    @Override
+    public Boolean isTrailingSlashMatch() {
+        return trailingSlashMatch == null ? Boolean.TRUE : trailingSlashMatch;
```

Comment:
And then have this Boolean.TRUE use ``Boolean.getValue(System.getProperty("myProperty", "true))`` as well (move that one-liner in a utility method)

Reply:
changed